src/components/NavigationBarAside.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+import NavigationBarAsideItems from './NavigationBarAsideItems';
+
+const NavigationBarAside = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
+
+  return (
+    <aside className="navigation-bar-aside">
+      <div className="navigation-bar-aside__wrapper">
+        <div
+          className={`navigation-bar-aside__profile-block ${
+            isLoggedIn ? 'logged-in' : 'logged-out'
+          }`}
+        >
+          {isLoggedIn ? (
```

Comment:
We won't need the `: null` part if we use `&&` 

```suggestion
          {isLoggedIn && (
```